client/packages.go

```diff
@@ -112,6 +114,42 @@ func (client *Client) List(names []string) ([]*Snap, error) {
 	return result, nil
 }
 
+func setPrice(snap *Snap, currency string) {
```

Comment:
I'd considered this but the suggested currency is required and it's currently only available as part of the `ResultInfo` returned from the API.